content/gocourse-2019-05-melbourne.slide

```diff
@@ -607,6 +607,17 @@ Before adding any commits to `feature` locally run:
 - Remove `Ready` `for` `review` and add `Under` `review`
 - Label with either `Approved` `by` `colleague` or `Changes` `requested` and remove `Under` `review`
 
+
+* Test samples
+
+- [[https://goplay.space/#HcgMVMtjWJo][Basics]]
+- [[https://goplay.space/#E29I7xTrz69][Multiple cases]]
+- [[https://goplay.space/#dCbjRkHqm_3][Testify]]
+- [[https://goplay.space/#XoXe-FUSoCD][Suites]]
+
+[[https://dave.cheney.net/2019/05/07/prefer-table-driven-tests][Dave Cheney - table driven tests]]
```

Comment:
Sorry I wasn't that clear- I think this link should set with the source code as a comment. (but you can also add it here)

Reply:
The article seemed like an interesting read if people want.